imcache_test.go

```diff
@@ -941,6 +942,53 @@ func TestImcache_GetAll_SlidingExpiration(t *testing.T) {
 	}
 }
 
+//GetAndRemove Tests Init
+func TestImcache_GetAndRemove(t *testing.T) {
```

Comment:
Could you also extend the `TestImcache_LongRunning_EvictionCallback` test to confirm the callback is really executed in a separate goroutine? See https://github.com/erni27/imcache/blob/master/imcache_test.go#L2198.